### PR TITLE
Suppress errors in suppression readme

### PR DESCRIPTION
**Pull request by sergey-shandar**
@vladbarosan Here's the real changes https://github.com/Azure/oav/pull/360/commits/f7be75c44f43631fd1ad917a563e8a957041c81d

Everything else is just restructure.

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -32,7 +32,8 @@ export {
   ValidationResult
 } from "./lib/util/validationError"
 
-export { executePromisesSequentially, clearCache } from "./lib/util/utils"
+export { executePromisesSequentially} from "./lib/util/utils"
```
Comment (vladbarosan) [inline index.ts]: why do we need to export this ?

Comment (sergey-shandar) [reply index.ts]: Because we run this function in `azure-rest-api-specs` CI. Sadly, directly from `node.js`.

Comment (sergey-shandar) [reply index.ts]: I mean `clearCache`. I have no idea about `executePromiseSequentially`. As soon as we get rid of API calls from portal and `azure-restapi-specs` CIs, I can throw away all of these exports.

Comment (vladbarosan) [reply index.ts]: not sure I follow this seems that it should be an internal function, only exported functions should be clear API calls like validate etc, but i saw now it is exported so I guess we have to stick with it. Not sure what you mean get rid of API calls, all node packages are in some sense libraries with APIs. sometimes they have a CLI wrapper, but thats irrelevant, at base they are libraries with an API

File: lib/util/jsonUtils.ts [hunk 2]
```diff
@@ -0,0 +1,121 @@
+import { Suppression, SuppressionItem } from '@azure/openapi-markdown'
```
Comment (vladbarosan) [inline lib/util/jsonUtils.ts]: can you point in these new files what was changed beyond splitting the old one please ?

**Pull request merged**